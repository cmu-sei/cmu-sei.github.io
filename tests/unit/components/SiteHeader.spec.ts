import { shallowMount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Component from '../../../components/SiteHeader.vue'

describe('SiteHeader.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Component)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
