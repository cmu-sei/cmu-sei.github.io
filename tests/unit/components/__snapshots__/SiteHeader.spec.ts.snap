// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SiteHeader.vue > matches snapshot 1`] = `
"<header class="bg-red-500">
  <anonymous-stub></anonymous-stub>
</header>"
`;
